packages/riipen-ui/src/components/VerticalProgressBar.test.jsx

```diff
@@ -0,0 +1,109 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import VerticalProgressBar from "./VerticalProgressBar";
+import VerticalProgressBarItem from "./VerticalProgressBarItem";
+
+describe("<VerticalProgressBar>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<VerticalProgressBar />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<VerticalProgressBar />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("renders given children", () => {
+      const child = <span>this is a child</span>;
+
+      const wrapper = mount(<VerticalProgressBar>{child}</VerticalProgressBar>);
+
+      expect(wrapper.find("VerticalProgressBar").contains(child)).toEqual(true);
```

Comment:
Could this be
```js
expect(wrapper.text()).to.include("this is a child");
```
Not sure if this works in this repo but noticed we're using `expect` instead of `assert`.

There are less implementation details here, ideally we don't want `.find("SomeComponent")` although in this case we're verifying that the whole child exists as a DOM node so it makes sense. Maybe it's actually possible to do
```js
expert(wrapper.contains(child)).toEqual(true);
```
I'm not sure, see which you prefer or leave it as is.

Reply:
Definitely works!  Updated to.include -> toContain and worked like a charm!